src/js/structs/Node.js

```diff
@@ -1,3 +1,5 @@
+import * as Status from "#PLUGINS/nodes/src/js/types/Status";
```

Comment:
Minor issue but this feels like a backwards dependency. Our node struct is in the main bundle and referenced by other main modules as well as placement and services plugins-- should the [Node]Status type also be there?

I don't know the correct answer to this.

Reply:
i also felt bad when i introduced this but then forgot about ironing it out.

we now pass a complete node to the status-module, so that it can figure out the status based on the information it gets from the node.